libraries/botbuilder-dialogs/src/dialogHelper.ts

```diff
@@ -64,6 +65,13 @@ export async function runDialog(
     await internalRun(context, dialog.id, dialogContext);
 }
 
+/**
+ * @param context The turn context.
+ * @param dialogId The dialog id.
+ * @param dialogContext The dialog context.
+ * @param dialogStateManagerConfiguration (optional) The dialog state manager configuration.
+ * @returns The dialogs turn result
+ */
 export async function internalRun(
```

Comment:
It should have not asked you to add docs to these functions, but now they have info, so, we probably just leave them like this